calico_containers/calico_ctl/node.py

```diff
@@ -122,6 +126,10 @@ def validate_arguments(arguments):
     if arguments.get("<DETACH>") or arguments.get("--detach"):
         detach_ok = arguments.get("--detach") in ["true", "false"]
 
+    detach_libnetwork_ok = (not detach_ok or
```

Comment:
if detch_ok is false, i.e. you've set it to some random value, then it doesn't make sense to make this check